guides/source/active_record_validations.md

```diff
@@ -991,9 +998,10 @@ class Invoice < ActiveRecord::Base
 end
 ```
 
-By default such validations will run every time you call `valid?`. It is also
-possible to control when to run these custom validations by giving an `:on`
-option to the `validate` method, with either: `:create` or `:update`.
+By default, such validations will run every time you call `valid?`
+or save the object. But it is also possible to control when to run these
+custom validations by giving an `:on` option to the `validate` method,
+with either: `:create` or `:update`.
```

Comment:
I don't like this change. It should be clear by now, that `save` and `create` call `valid?` before actually saving the record. This is just duplicating that fact. The essence here is `valid?` and that the reader understand what methods will call `valid?`.